src/App.js

```diff
@@ -5,25 +5,27 @@ import UserOutput from './UserOuput/UserOutput';
 
 class App extends React.Component {
   state = {
-    username: 'Olyaaaa',
+    text: 'Olyaaaa',
   }
 
-  userInputHandler = (event) => {
-    if (event.key === "Enter") {
-      this.setState({
-        username: event.target.value,
-      })
-    }
+  handleTextInput = (event) => {
+    this.setState({
+      text: event.target.value,
+    })
   }
 
-  render() {
-    const size = 2;
-    const outputElements = Array(size).fill( <UserOutput username={this.state.username}/>);
+  handleLetterDeletion = (index) => {
+    const text = [...this.state.text];
+    text.splice(index, 1)
+
+    this.setState({text: text.join('')});
+  }
 
+  render() {
     return (
       <div className="App">
-        <UserInput onKeyPress={this.userInputHandler} username={this.state.username} />
-        {outputElements}
+        <UserInput onChange={this.handleTextInput} text={this.state.text} />
```

Comment:
You are using `this.state.text` twice, so 
```
cosnt {text} = this.sate
```
in the begginig of ` render()` make sense

Reply:
done